package.json

```diff
@@ -23,11 +23,13 @@
     "@mantine/next": "^4.2.4",
     "@mantine/notifications": "^4.2.10",
     "@mantine/prism": "^4.2.11",
+    "@testing-library/user-event": "^14.2.1",
```

Comment:
I kept `userEvent` instead of `fireEvent` because I thought that `fireEvent.change()` couldn't simulate typing because CodeMirror doesn't have `target.value` property. I could def be wrong though, should I still change it to fireEvent?

Reply:
Ughh testing with codemirror is a nightmare. I guess it should be fine to leave for now since the code that uses it is "frozen" until further notice anyway. Hoping we ultimately can accomplish it with fireEvent if we can drill down to the correct component, but you're probably right :(